app/assets/stylesheets/content/_index.scss

```diff
@@ -74,6 +74,10 @@
   .filter-form {
     @include govuk-responsive-padding(3, "left");
 
+    .govuk-label--s {
```

Comment:
If this needs to be smaller than the `--s` size of a label (which is currently `19px`) shall we define this as a `.app-label--xs` to know that it's smaller than the `--s` and it's being defined at the app level?

I do get this is an overwrite, being applied to the hints too, where you most probably won't be able to pass a class right now, so I get if this is not possible at the moment.